micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jpa/HibernateQueryMetrics.java

```diff
@@ -113,49 +112,46 @@ public void onPostLoad(PostLoadEvent event) {
         void registerQueryMetric(Statistics statistics) {
             for (String query : statistics.getQueries()) {
                 QueryStatistics queryStatistics = statistics.getQueryStatistics(query);
-                if (queryStatistics == null) continue;
-                String queryName = meterRegistry.config().namingConvention().tagValue(query);
```

Comment:
Removing this removes sanitization of the query tag value, no?

Reply:
@shakuzen I thought that it's okay as we already sanitize tag values in meter registries as we do for other meters.

Reply:
That's true. We usually register names/values in a canonical form also, but you're right it doesn't make sense to do double work here.